pkg/services/authz/zanzana/schema/schema_core.fga

```diff
@@ -0,0 +1,32 @@
+module core
+
+type namespace
+  relations
+    define view: [user, team#member, role#assignee] or edit
+    define edit: [user, team#member, role#assignee] or admin
+    define admin: [user, team#member, role#assignee]
+
+    define read: [user, team#member, role#assignee] or view
+    define create: [user, team#member, role#assignee] or edit
+    define write: [user, team#member, role#assignee] or edit
+    define delete: [user, team#member, role#assignee] or edit
+    define permissions_read: [user, team#member, role#assignee] or admin
+    define permissions_write: [user, team#member, role#assignee] or admin
+
+type user
+
+type role
+  relations
+    define assignee: [user, team#member, role#assignee]
+
+type team
+  relations
+    # Action sets
+    define admin: [user]
+    define member: [user] or admin
+
+    define read: [role#assignee] or member
+    define write: [role#assignee] or admin
+    define delete: [role#assignee] or admin
+    define permissions_read: [role#assignee]
```

Comment:
so we don't grant admins permissions_read automatically?

Reply:
We should :)